styles/globals.css

```diff
@@ -2,12 +2,18 @@
 @tailwind components;
 @tailwind utilities;
 
+@font-face {
+  font-family: 'NeuzeitSLTStd';
+  src: url('../public/fonts/NeuzeitSLTStd-Book.otf');
```

Comment:
Not sure whether we should also add the [heavy font file](https://www.download-free-fonts.com/details/79166/neuzeit-s-lt-std-book-heavy) for the bold version or not 🤔 

And there seems to be an issue with Figma which when we don't install the font on our machine we'll see the value of `font-weight` for bold text differently from when we installed them